x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/500_rrf_explain.yml

```diff
@@ -0,0 +1,386 @@
+setup:
```

Comment:
No tests with retrievers?

Reply:
Yeap, for some reasons I did not include any :/ Will add some similar tests but written using retrievers :)

Reply:
Added :)